website/docs/rbi.md

```diff
@@ -151,42 +155,79 @@ a method's arity, they can also declare a method's input and output types.
+When initializing a new Sorbet project, `tapioca init` reads the project's
+`Gemfile`, checks to see if `rbi-central` already has any suitable RBI files,
 and fetches them into the current project if so. After initialization, to update
-or add new RBI files from `sorbet-typed`, run:
+or add new RBI files from `rbi-central`, run:
 
 ```bash
-srb rbi sorbet-typed
+❯ bin/tapioca annotations
 ```
 
-When `sorbet-typed` does not have RBI files for a gem, `srb init` will still
+When `rbi-central` does not have RBI files for a gem, `tapioca init` will still
 have created some autogenerated RBI files for that gem. These autogenerated
 files are a **great** way to start off writing hand-written RBIs for a gem!
-Simply move the autogenerated file out of the RBI `sorbet/rbi/sorbet-typed/`
-folder, and start modifying it by hand.
+Simply copy the autogenerated file from the `sorbet/rbi/gems/` folder to
+`sorbet/rbi/shims/gems`, and start modifying it by hand.
 
 ### RBIs within gems
 
 We expect that as adoption grows, gems will include their external interfaces
-RBI files in an `rbi/` directory. When they do this, Sorbet will automatically
-include those definitions when typechecking a project. In the future, we
-anticipate this to be the preferred way to include RBI files into a project.
-
-`srb tc` will automatically check the `Gemfile.lock` for changes since it was
-last run. If there are changes (or if it has never run with this specific hash
-before) it will check every gem to see if they have a `rbi/` directory. If so,
-they will all be written out to a `~/.cache/sorbet/gem-rbis/<hash>` file and
-used when typechecking the code. These are not included in the normal `sorbet/`
-files because they will be machine dependent so we don't want those ending up in
-version control.
+RBI files in an `rbi/` directory. When they do this, Tapioca will automatically
+merge those definitions in the autogenerated RBIs. In the future, we anticipate
+this to be the preferred way to include RBI files into a project.
+
+## DSL RBIs
+
+Sorbet by itself does not understand DSLs involving meta-programming, such as
+Rails. This means that Sorbet won't know about constants and methods generated
+by `ActiveRecord` or `ActiveSupport`. To solve this, Tapioca can load your
+application and introspect it to find the constants and methods that would exist
+at runtime and compile them into RBI files.
+
+To generate the RBI files for the DSLs used in your application:
+
+```bash
+# Update the autogenerated DSL RBIs:
+❯ bin/tapioca dsl
+
+# Update the autogenerated DSL RBI for a specific constant:
+❯ bin/tapioca dsl <ConstantName>
+```
+
+You can read about all the DSL RBI compilers supplied by Tapioca in the
+[Tapioca's manual](https://github.com/Shopify/tapioca/blob/main/manual/compilers.md).
 
 ## The Hidden Definition RBI
 
+> **The `srb rbi hidden-definitions` is in
+> [maintenance mode](../blog/2022/07/28/srb-tapioca), please use `tapioca`
```

Comment:
Is there a better way to link to blog posts?

Reply:
I think this is likely best.